app/code/Magento/ImportExport/Ui/DataProvider/ExportFileDataProvider.php

```diff
@@ -67,6 +76,9 @@ public function __construct(
             $meta,
             $data
         );
+
+        $this->fileIO = $fileIO ?: ObjectManager::getInstance()->get(File::class);
+        $this->request = $request;
```

Comment:
Sorry, just spotted one more point. Do we need this $request assignment here? I'm asking because the parent class has the protected request property (https://github.com/magento/magento2/blob/c4eeaa9a827c533e46f35af5cf7e10f8eb4737b3/lib/internal/Magento/Framework/View/Element/UiComponent/DataProvider/DataProvider.php#L100), so we can access it in the child class without the assignment.

Reply:
Hi @rogyar you're right. Just removed it.
Thank you.